android/BOINC/app/src/main/java/edu/berkeley/boinc/PrefsFragment.java

```diff
@@ -79,7 +78,8 @@ public void onReceive(Context context,Intent intent) {
 	@Override
 	public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
     	if(Logging.VERBOSE) Log.d(Logging.TAG,"ProjectsFragment onCreateView");
-        // Inflate the layout for this fragment
+
+    	// Inflate the layout for this fragment
     	View layout = inflater.inflate(R.layout.prefs_layout, container, false);
 		lv = layout.findViewById(R.id.listview);
```

Comment:
Would you mind fixing these two line WRT their indentation? Thanks 👍

Reply:
Done.